### ISSUE TITLE
Add missing rust docs

### ISSUE THREAD

**Issue by notmandatory** [bitcoindevkit/rust-electrum-client#82]
To find list of missing rust docs see: 
```shell
% cargo build --all-features
   Compiling electrum-client v0.10.3 (/Users/steve/git/notmandatory/rust-electrum-client)
warning: missing documentation for a struct
 --> src/config.rs:5:1
  |
5 | pub struct Config {
  | ^^^^^^^^^^^^^^^^^
  |
note: the lint level is defined here
 --> src/lib.rs:1:9
  |
1 | #![warn(missing_docs)]
  |         ^^^^^^^^^^^^

warning: missing documentation for an associated function
   --> src/config.rs:101:5
    |
101 |     pub fn socks5(&self) -> &Option<Socks5Config> {
    |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

warning: missing documentation for an associated function
   --> src/config.rs:104:5
    |
104 |     pub fn retry(&self) -> u8 {
    |     ^^^^^^^^^^^^^^^^^^^^^^^^^

warning: missing documentation for an associated function
   --> src/config.rs:107:5
    |
107 |     pub fn timeout(&self) -> Option<Duration> {
    |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

warning: missing documentation for an associated function
   --> src/config.rs:110:5
    |
110 |     pub fn validate_domain(&self) -> bool {
    |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

warning: missing documentation for an associated function
   --> src/config.rs:114:5
    |
114 |     pub fn builder() -> ConfigBuilder {
    |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

warning: missing documentation for a type alias
  --> src/types.rs:20:1
   |
20 | pub type Call = (String, Vec<Param>);
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

warning: `electrum-client` (lib) generated 7 warnings
```

**Comment by sansosti** [comment]
Hi there,
  I was looking for issues to contribute with but I think this one can be closed now?

**Comment by oleonardolima** [comment]
Closed by #94
